plugins/versionpress/tests/Automation/WpAutomation.php

```diff
@@ -557,12 +557,17 @@ private function prepareFiles()
     private function ensureCleanInstallationIsAvailable()
     {
 
-        if (!is_dir($this->getCleanInstallationPath())) {
-            $downloadPath = $this->getCleanInstallationPath();
-            FileSystem::mkdir($downloadPath);
+        $cleanInstallationPath = $this->getCleanInstallationPath();
+
+        if (!is_dir($cleanInstallationPath) || !is_file($cleanInstallationPath . '/wp-settings.php')) {
+            if (!is_dir($cleanInstallationPath)) {
```

Comment:
Yes, that looks better. I also agree with extracting the check method. Will do that.